library/cartridge_connect_to_membership.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+
+import math
+import pkgutil
+
+if pkgutil.find_loader('ansible.module_utils.helpers'):
+    import ansible.module_utils.helpers as helpers
+else:
+    import module_utils.helpers as helpers
+
+argument_spec = {
+    'console_sock': {'required': True, 'type': 'str'},
+    'hostvars': {'required': True, 'type': 'dict'},
+    'play_hosts': {'required': True, 'type': 'list'},
+}
+
+BATCH_SIZE = 32
+
+
+def connect_instances(control_console, play_hosts, instance_names, advertise_uris):
+    connect_errors, _ = control_console.eval_res_err('''
+        local advertise_uris = ...
+
+        local cartridge = require('cartridge')
+        local fiber = require('fiber')
+
+        local count = 0
+        local channel = fiber.channel()
+
+        for _, advertise_uri in ipairs(advertise_uris) do
+            count = count + 1
```

Comment:
Can we skip instances that are already in membership?

```lua
if membership.get_member(advertise_uri) == nil then
    count = count + 1 
....
end
```

Reply:
Yes, I added this check